rsconf/package_data/rsconf/rsconf.sh

```diff
@@ -587,9 +608,9 @@ rsconf_yum_install() {
             todo+=( "$x" )
         fi
     done
-    declare cmd=install
-    if [[ ${rsconf_yum_reinstall:-} ]]; then
-        cmd=reinstall
+    declare cmd="${rsconf_yum_install_cmd:-install}"
+    if [[ ! $cmd =~ ^((re)?install|downgrade)$ ]]; then
```

Comment:
In general, we don't write code like this, but it does make the code clearer. I guess I'm just lazy. :)

For the error messages (below), we have been using "<msg> <name>=<val>" format. The traditional shell "<val>: <msg>" is suboptimal when there are multiple values with errors. In this case include `rsconf_yum_install_cmd` so we know what the problem is, e.g.  `unexpected value rsconf_yum_install_cmd=$cmd`.

Reply:
f44cd01